pkg/db/db_local/install.go

```diff
@@ -40,7 +40,7 @@ If you need to restore the contents of your public schema, please open an issue
 
 // EnsureDBInstalled makes sure that the embedded pg database is installed and running
 func EnsureDBInstalled(ctx context.Context) (err error) {
-	utils.LogTime("db_local.EnsureDBInstalled start")
+	utils.LogTime("db_local.EnsureDBInstallation start")
```

Comment:
EnsureDBInstallation  ???
Function is still ``EnsureDBInstalled`

Reply:
done